content/intro/overview.md

```diff
@@ -0,0 +1,34 @@
+---
+linkTitle: अवलोकन
+title: Brigade की अवलोकन
+description: Brigade का उच्च स्तरीय दृश्य
+section: परिचय
+weight: 1
+aliases: 
+  - /overview.md
+  - /intro/overview.md
+---
+
+Brigade एक इवेंट संचालित स्क्रिप्टिंग टूल है जो Kubernetes पर चलता है। इसका मतलब यह है:
+
+- Brigade क्लाउड में स्क्रिप्ट योग्य स्वचालित कार्यों को चलाने के लिए है।
+- Brigade के लिए आपको कोई होस्ट सर्वर प्रबंधित करना ज़रुरी नहीं है।
+-Brigade विशेष रूप से सीआई और सीडी वर्कलोड के लिए उपयुक्त है जैसे:
+- Automated testing
+  - GitHub hook integration
+  - Building artifacts and releases
+  - Managing deployments
+- Brigade अपने API server के साथ रहता है |
+  - बातचीत सीधे ब्रिगेड के माध्यम से होती है, `brig` CLI या SDK के माध्यम से
+  - ब्रिगेड स्थापना का प्रबंधन करने वाले ऑपरेटर अभी भी ब्रिगेड संसाधनों की जांच के लिए K8s API पर जा सकते हैं।
+  - ब्रिगेड सर्वर को हेल्म के साथ तैनात और प्रबंधित किया जाता है, इस प्रकार होस्टिंग किसी भी स्टॉक कुबेरनेट्स सिस्टम पर हो सकती है, एज़्योर से मिनिक्यूब तक।
+- Brigade authn/authz की चिंता को संभलता है |
+  - तृतीय-पक्ष प्रमाणीकरण विकल्पों में GitHub का OAuth2 पहचान प्रदाता और OIDC- अनुरूप समाधान जैसे Azure Active Directory और Google Identity Platform शामिल हैं।
```

Comment:
Same comment for English words.

Reply:
What should I write these in Hindi?

Reply:
One option is to transliterate the proper nouns. 

For example,
1. `GitHub` becomes `गिटहब (GitHub)`
2. `Azure Active Directory` becomes `ऐज़ूर एक्टिव डायरेक्टरी (Azure Active Directory)`
3. `OAuth2` becomes `औ ौथ २ (OAuth2)`

In this case, the first occurrence always has an English version too.

The second option is to do the other way, i.e.

For example,
1. `GitHub` becomes `GitHub (गिटहब)`
2. `Azure Active Directory` becomes `Azure Active Directory (ऐज़ूर एक्टिव डायरेक्टरी)`
3. `OAuth2` becomes `OAuth2 (औ ौथ २)`

This will be easier to maintain. Thoughts?

Reply:
Okay, I have written many of such words (computer programming related) in English.

Reply:
Yup, we will need to decide on a format. I believe the second option is better.

Reply:
Yeah, 2nd one seems better. I can start doing it , but it may take sometime because a lot has to be changed.

Reply:
Yes, please take your time!